src/Microsoft.AspNet.OData/Formatter/ODataModelBinderProvider.cs

```diff
@@ -23,6 +23,10 @@ namespace Microsoft.AspNet.OData.Formatter
     /// <summary>
     /// Provides a <see cref="IModelBinder"/> for EDM primitive types.
     /// </summary>
+    /// <remarks>
+    /// This class is similar to ODataModelBinder in AspNetCore. The flow is similar but the
```

Comment:
File located in AspNet project, you write AspNetCore

Reply:
Correct., my intention is to cross-link the files since a fix would likely need to make it into both files.